index.html

```diff
@@ -36,8 +35,142 @@ <h2 class="subtitle">I'm a Software Developer</h2>
             </div>
         </section>
 
-    </main>
+        <section class="workssection">
+            <div class="worksstart">
+                <h1 class="workstitle">My recent works</h1>
```

Comment:
- [Optional] please use the exact text as in the Figma, for now. expected to be `My Recent Works` not `My recent works`, as using the one on the Figma adds more aesthetics to your project design. 

##### Your design 
![Screenshot from 2022-05-27 18-05-15](https://user-images.githubusercontent.com/58771507/170749335-cd110073-a30f-4e4d-a37f-66a7de749fbe.png)

##### Figma design
![Screenshot from 2022-05-27 18-06-34](https://user-images.githubusercontent.com/58771507/170750120-21cbc897-9193-4a75-8478-440ff44b4705.png)